pkg/cloudprovider/aws/cloudprovider.go

```diff
@@ -154,8 +154,8 @@ func (c *CloudProvider) GetInstanceTypes(ctx context.Context) ([]cloudprovider.I
 	return c.instanceTypeProvider.Get(ctx)
 }
 
-func (c *CloudProvider) Terminate(ctx context.Context, nodes []*v1.Node) error {
-	return c.instanceProvider.Terminate(ctx, nodes)
+func (a *CloudProvider) Terminate(ctx context.Context, node *v1.Node) error {
```

Comment:
Why are changing the receiver from c -> a?

Reply:
I don't know how this happened. Good catch.